.pre-commit-config.yaml

```diff
@@ -14,3 +14,18 @@ repos:
       - id: check-json
       - id: detect-aws-credentials
         args: [ --allow-missing-credentials ]
+  - repo: https://github.com/pycqa/pydocstyle
+    rev: 6.1.1
+    hooks:
+      - id: pydocstyle
+        additional_dependencies: [ "toml" ]
+  - repo: local
+    hooks:
+      - id: pyright
+        name: pyright
+        entry: pyright
+        language: node
+        files: ^backend/
```

Comment:
does this not need to be `^src/`?

Reply:
whoops! it seemed to work anyway but i've changed it 😀